packages/components/src/Loading/LoadingParagraph/LoadingParagraph.module.scss

```diff
@@ -0,0 +1,50 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "../mixins";
+
+.loadingParagraph {
+  @extend %loadingBase;
+
+  background-color: $color-gray-300;
+  height: calc(#{$spacing-md} * 0.625);
+  margin-bottom: calc(#{$spacing-md} * 0.5);
```

Comment:
optional knit:  we could change this to  `$spacing-12`. Unfortunately nothing to do with the gnarly one above 🥲 We could also swap out all the spacing-md's but not sure how much that would help the rando calcs

Reply:
Nice catch! I won't fix it now, but certainly worth updating the spacing tokens later. I'll put something in the Jira.